packages/@sfx/sayt-driver-plugin/test/unit/common/sayt-driver-plugin.test.ts

```diff
@@ -51,6 +52,25 @@ describe('Sayt Driver Plugin', () => {
     };
   });
 
+  describe('constructor()', () => {
+    it('should save a passed product transformer as a transformProduct property', () => {
+      const productTransformer = (product) => 123;
+
+      const driver = new SaytDriverPlugin({ productTransformer });
+
+      expect(driver.transformProduct).to.equal(productTransformer);
+    });
+
+    it('should have a default identity transformProduct property if no transformer is passed', () => {
+      const driver = new SaytDriverPlugin();
+      const object = { some: 'object' };
```

Comment:
It would be better to annotate this as `any` instead of casting it where it's used.

```suggestion
      const object: any = { some: 'object' };
```